models/Encode Data Model.html

```diff
@@ -0,0 +1,257 @@
+<HTML>
```

Comment:
@kreinold do we need to check in the `cmap` and `html`, or are the `ttl` files enough to regenerate all the info?